CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [BUGFIX] Querier: Streaming remote read was returning 1 chunk per frame after the first frame of a series. #4423
```

Comment:
This should be sorted with the other `[BUGFIX]` tags at the bottom of the changelog file. Can you clarify the comment a bit, something like "Streaming read will return multiple chunks per frame after the first frame."?

Reply:
Oop, didn't notice they were sorted--fixed!